Add Application Connector Tests (#4220)
kyma-project/kyma: tests/application-connector-tests/README.md

# Application Connector Tests

## Overview

This project contains the acceptance tests that you can run as part of the Kyma Application Connector testing process.

## Usage

Environment parameters used by the tests:

| Name | Required | Default | Description | Possible values |
|------|----------|---------|-------------|-----------------|

> | **NAMESPACE** | Yes | - | The Namespace in which the test Application will operate | `kyma-integration` |

Reviewer: ```suggestion
| **NAMESPACE** | Yes | - | The Namespace in which the test Application runs. | `kyma-integration` |
```

> | **CENTRAL** | No | false | Determines if the Connector Services acts as a Central | true | 

Reviewer: ```suggestion
| **CENTRAL** | No | false | Determines if the Connector Service operates in the central mode.  | true | 
```

> | **SKIP_SSL_VERIFY** | No | false | Determines if the TLS should be skipped | true | 

Reviewer: ```suggestion
| **SKIP_SSL_VERIFY** | No | false | Determines if the TLS should be skipped. | true | 
```

### Run locally

> If the test is not able to find `InClusterConfig`, it will try to use the local `kubeconfig` file.

Reviewer: ```suggestion
If the test can't find `InClusterConfig`, it uses the local `kubeconfig` file.
```

Reviewer: Is `InClusterConfig` a file? If so, the sentence should be: 
> If the test can't find the `InClusterConfig` file, it uses the local`kubeconfig` file.

> To run tests locally export required environment variables:

Reviewer: ```suggestion
To run tests locally, export the required environment variables:
```

```
export NAMESPACE=kyma-integration
export CENTRAL=false
export SKIP_SSL_VERIFY=true
```
 

> And use `go test` to run the tests:

Reviewer: ```suggestion
Use `go test` to run the tests:
```
